components/RepeatDelayed.coffee

```diff
@@ -14,22 +14,21 @@ exports.getComponent = ->
     datatype: 'number'
     description: 'How much to delay'
     default: 500
+    control: true
 
   c.outPorts.add 'out',
     datatype: 'all'
 
-  noflo.helpers.WirePattern c,
-    in: 'in'
-    params: 'delay'
-    out: 'out'
-    forwardGroups: true
-    async: true
-  , (payload, groups, out, callback) ->
-    timer = setTimeout =>
-      out.send payload
-      do callback
+  c.process (input, output) ->
+    return unless input.has 'in'
+    delay = input.getData 'delay'
```

Comment:
Is `InPort.default` playing nicely with Process API here? I didn't test it in Component spec.